pages/api/request/support.ts

```diff
@@ -5,9 +5,9 @@ import { logMessage } from "@lib/logger";
 
 // Allows an authenticated or unauthenticated user to send an email requesting help
 const requestSupport = async (req: NextApiRequest, res: NextApiResponse) => {
-  const { supportType, name, email, request, description } = req.body;
+  const { supportType, name, email, request, description, department, branch, jobtitle } = req.body;
 
-  if (!name || !email || !request || !description) {
+  if (!name || !email || !request || !description || !department || !branch || !jobtitle) {
     return res.status(404).json({ error: "Malformed request" });
```

Comment:
Not an issue with your PR, but is there a reason we're doing 404s for this error? Seems like 400 would be more appropriate, no?

Reply:
Missed that, good point